src/redux/Graphql/saga.js

```diff
@@ -53,6 +194,10 @@ export function* getTopicsHandler() {
         value: [],
         limit: action.limit,
         skip: action.skip,
+        error: {
```

Comment:
these background graphql calls dont need error objects since we fail silently, remember?